MyOnboarding/src/MyOnboardingApp.Api/Models/TodoListItem.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace MyOnboardingApp.Api.Models
+{
+    [DataContract]
```

Comment:
You don't have to specify DataContract and DataMember as it used in WCF. We are using [WebApi](https://stackoverflow.com/questions/43775132/difference-between-wcf-web-api-wcf-rest-and-web-service), there is no need to specify anything for the REST methods.